packages/veto-app/frontend/src/api/gun.ts

```diff
@@ -0,0 +1,5 @@
+import Gun from 'gun/gun'
+
+export default Gun({
+  peers: [`http://${window.location.hostname}:8080/gun`],
```

Comment:
Is `8080` the port we will use in production?

It would be better to keep that port in `config.ts` so we can easily change it

Reply:
Changed to use the `config`